swerve100/src/main/java/org/team100/lib/motor/turning/CANTurningMotor.java

```diff
@@ -66,12 +61,19 @@ public void set(double output) {
         m_motor.set(output);
     }
 
-    public void setPID(ControlMode control, double output) {
-        int rotationsPerRev = 1666;
-        double goal = m_motor.getSelectedSensorPosition()
-                + MathUtil.angleModulus(2 * Math.PI * (output - m_motor.getSelectedSensorPosition() / rotationsPerRev))
-                        / (2 * Math.PI) * rotationsPerRev;
-        m_motor.set(control, goal);
+    public void setPIDVelocity(double outputRadiansPerSec) {
+        double ticksPerRevolution = 1024;
+        double revolutionsPerSec = outputRadiansPerSec/(2*Math.PI);
+        double revsPer100ms = revolutionsPerSec/10;
+        double ticksPer100ms = revsPer100ms*ticksPerRevolution;
+        m_motor.set(ControlMode.Velocity, ticksPer100ms);
+    }
+
+
+    public void setPIDPosition(double outputRadians) {
```

Comment:
does anything actually use this?  if you're not using it you can just remove it.